### PR TITLE
docs(readme): add usage instructions and examples

### PR DESCRIPTION
**Pull request by elnaz**
Might be easier to read in [this mode](https://github.com/lob/hapi-bookshelf-total-count/blob/readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,148 @@
...
+    }
+  }
+});
+
+```
+
+### Request
+
+```bash
+$ curl -g GET "https://YOUR_DOMAIN/books?include[]=total_count"
```
Comment (charlesbjohnson) [inline README.md:51]: Is the `-g` necessary? I thought that if the `[]` is in quotes then it should be fine.

**Comment by charlesbjohnson** [comment]
:+1: \* :100:

File: README.md (hunk 1 above)
Comment (elnaz) [inline README.md:51]: I thought so too but I'm getting a globbing error when the brackets are inside quotes and the `-g` is not present

**Comment by mgartner** [comment]
LGTM

**Pull request merged**